### PR TITLE
Search, first iteration

### PR DESCRIPTION
**Pull request by amdaddeo**

### CODE REVIEW

File: app/controllers/Application.java [hunk 1]
```diff
@@ -13,31 +17,27 @@
...
   }
 
   @Before
   static void addDefaults() {
-  }
-
-  public static User user() {
-    assert Secure.Security.connected() != null;
-    return User.find("byEmail", Secure.Security.connected()).first();
+          renderArgs.put("searched", Users.searchForUser("bob").name);
```
Comment (amshali) [inline app/controllers/Application.java:26]: What is this doing in addDefaults? You should not do that.

File: app/controllers/Application.java [hunk 2]
```diff
@@ -13,31 +17,27 @@
   @Before
   static void setConnectedUser() {
     if (Security.isConnected()) {
-      renderArgs.put("currentUser", user());
+      renderArgs.put("currentUser", Users.user());
```
Comment (amshali) [inline app/controllers/Application.java:20]: Let the user() function be in in OBController. Don't move it to Users.

File: conf/routes [hunk 3]
```diff
@@ -5,6 +5,10 @@
 # Import Secure routes
 *       /                                       module:secure
 
+GET     /users/search                           UserSearch.search
+POST    /users/search                           UserSearch.search
```
Comment (amshali) [inline conf/routes:9]: Why POST and GET?

**Comment by amshali** [comment]
Please fix the issue and send a pull request again.

File: app/controllers/Application.java (hunk 2 above)
Comment (amdaddeo) [inline app/controllers/Application.java:20]: What is the OBController?

File: conf/routes (hunk 3 above)
Comment (amdaddeo) [inline conf/routes:9]: The example code had POST but I guess GET is all the search is using at this point.

File: app/controllers/Application.java (hunk 2 above)
Comment (amshali) [inline app/controllers/Application.java:20]: It's the Super Controller for OpenBook. Your controller must extend that.

File: conf/routes (hunk 3 above)
Comment (amshali) [inline conf/routes:9]: Can you remove one?

File: app/controllers/Application.java (hunk 2 above)
Comment (amdaddeo) [inline app/controllers/Application.java:20]: OK, moved User back to Application

File: conf/routes (hunk 3 above)
Comment (amdaddeo) [inline conf/routes:9]: Changed to only GET

File: app/controllers/Application.java (hunk 1 above)
Comment (amdaddeo) [inline app/controllers/Application.java:26]: Removed this.

File: app/controllers/Application.java (hunk 2 above)
Comment (amshali) [inline app/controllers/Application.java:20]: user() is in OBController, it doesn't have to be in Application. Look at the latest repo.

**Pull request closed without merging**